lib/Command/PhoneNumber/AddPhoneNumber.php

```diff
@@ -0,0 +1,105 @@
+<?php
+
+declare(strict_types=1);
+/**
+ * SPDX-FileCopyrightText: 2025 Nextcloud GmbH and Nextcloud contributors
+ * SPDX-License-Identifier: AGPL-3.0-or-later
+ */
+
+namespace OCA\Talk\Command\PhoneNumber;
+
+use OC\Core\Command\Base;
+use OCA\Talk\Model\PhoneNumber;
+use OCA\Talk\Model\PhoneNumberMapper;
+use OCP\AppFramework\Db\DoesNotExistException;
+use OCP\IPhoneNumberUtil;
+use OCP\IUser;
+use OCP\IUserManager;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+
+class AddPhoneNumber extends Base {
+
+	public function __construct(
+		private IUserManager $userManager,
+		private IPhoneNumberUtil $phoneNumberUtil,
+		private PhoneNumberMapper $mapper,
+	) {
+		parent::__construct();
+	}
+
+	#[\Override]
+	protected function configure(): void {
+		$this
+			->setName('talk:phone-number:add')
+			->setDescription('Add a mapping entry to map a phone number to an account')
+			->addArgument(
+				'phone',
+				InputArgument::REQUIRED,
+				'Phone number that will be called',
+			)
+			->addArgument(
+				'account',
```

Comment:
```suggestion
				'user',
```

Other occ commands use `user`. For occ consistency I would keep with that.